internal/app/app.go

```diff
@@ -38,6 +48,13 @@ const (
 	KVProcessorContext           = "kv_processor"
 )
 
+// retries configuration for fetching connection info
+var (
+	RtyAtt = retry.Attempts(uint(5))
+	RtyDel = retry.Delay(time.Millisecond * 10000)
```

Comment:
1. see no reasons for these variables to be public
2. `time.Millisecond * 10000` can be replaced with `time.Second * 10`